bin/macro-benchmark.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+let fs = require('fs-extra');
+let path = require('path');
+
+let args = process.argv;
+let [ command, ...opts ] = args.slice(2);
```

Comment:
I'd suggest something like `yargs` here instead of doing it manually. At the moment, this is fairly easy but as we continue to add commands (and those commands start having their own args/options/etc) it will get harder and harder.

* http://yargs.js.org/
* https://github.com/yargs/yargs

Example with multiple sub-commands, options, help output, etc: [here](https://github.com/rwjblue/gather-static-template-strings/blob/master/src/cli.js).